manifests/b/BlenderFoundation/Blender/4.0.0/BlenderFoundation.Blender.installer.yaml

```
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: BlenderFoundation.Blender
+PackageVersion: 4.0.0
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+FileExtensions:
+- 3ds
+- fbx
+- glb
+- gltf
+- obj
+- ply
+- stl
+ProductCode: '{B557A37F-811D-4A98-9306-FB34E069D999}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://ftp.nluug.nl/pub/graphics/blender/release/Blender4.0/blender-4.0.0-windows-x64.msi

```

```suggestion
  InstallerUrl: https://www.blender.org/download/release/Blender4.0/blender-4.0.0-windows-x64.msi
```
I believe this URL will redirect the user to the closest mirror.